src/modules/job-manager/kill.c

```diff
@@ -38,7 +38,11 @@
 #include "job-manager.h"
 
 #ifndef SIGRTMAX
```

Comment:
Or just use NSIG.

That's what I did before in d51397b3ecfd13a1e93ad50a40f80c1d5cc20bb0